ref/api.md

```diff
@@ -506,6 +507,38 @@ Bugs and limitations:
     `hasTrailingZero` above, and
     [#178](https://github.com/scottlamb/moonfire-nvr/issues/178).
 
+### `DELETE /api/cameras/<uuid>/<stream>/view.mp4`
+
+Requires the `viewVideo` permission.
```

Comment:
I think we should make a new permission, maybe `adminStorage`? It'd be really surprising if a `view...` permission allowed folks to destroy something, and I really do want to allow folks to view without deleting.